src/main/webapp/definitions/alfresco-core.yaml

```diff
@@ -11027,6 +11065,45 @@ definitions:
          format: date-time
       values:
         type: object
+  ActionConstraint:
+    description: |
+      Response object holding action parameter constraint name and map of its values-labels.
+    type: object
+    properties:
+      name:
+        type: string
+        description: |
+          Name of the constraint.
+      constraintsMap:
+        description: |
+          A map (String-String) of constraint values and labels.
```

Comment:
Do you happen to have an example of how this works in the v0 API?  I can't think how we can handle all the different constraints using a String-String map.

Reply:
This one isn't very intuitive. Constraint value is a key of the map, whereas the label is the value of the map. Constraint values are fixed and labels are translatable. See the email I sent you with example responses - maybe that will help a bit.